src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -838,4 +842,32 @@ def load_arguments(self, _):
     with self.argument_context('vm monitor log show') as c:
         c.argument('analytics_query', options_list=['--analytics-query', '-q'], help="Query to execute over Log Analytics data.")
         c.argument('timespan', help="Timespan over which to query. Defaults to querying all available data.")
+
+    with self.argument_context('vm monitor metrics') as c:
+        c.argument('metricnamespace', options_list=['--namespace'],
+                   help='Namespace to query metric definitions for.')
+
+    with self.argument_context('vm monitor metrics tail') as c:
+        from azure.mgmt.monitor.models import AggregationType
```

Comment:
Because monitor supports multiapi, please use get_models() to import the corresponding models.

Reply:
monitor doesn't support multiapi now. I will fix it and monitor part in another PR.